### PR TITLE
global gene search help text  (SCP-2712)

### PR DESCRIPTION
**Pull request by devonbush**
Now updated with the discussion from Friday.  Shorter placeholder text, and a tag select-style widget (react-select, which is the same widget terra uses).  There may be remaining CSS tweaks and/or text polishing, but I think we can merge this now to make sure we have plenty of time to test the base functionality on staging.

image

image

### CODE REVIEW

File: app/javascript/components/search/genes/GeneSearchView.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export default function GeneSearchView() {
                                  !geneSearchState.isLoading &&
                                  !geneSearchState.isError
 
-  let geneSearchPlaceholder = 'Search for genes across all studies'
+  let geneSearchPlaceholder = 'genes (e.g. "agpat pten")'
```
Comment (eweitz) [inline app/javascript/components/search/genes/GeneSearchView.js:30]: Nit: title case is SCP and DSP convention.

```suggestion
  let geneSearchPlaceholder = 'Genes (e.g. "agpat pten")'
```

**Review by eweitz — APPROVED** [review]
Looks good!  Thanks for openness to and quick integration of discussion feedback.

**Comment by eweitz** [comment]
Please append Jira ticket to title.

File: app/javascript/components/search/genes/GeneSearchView.js (hunk 1 above)
Comment (eweitz) [inline app/javascript/components/search/genes/GeneSearchView.js:30]: [Searching genes for AGPAT](https://singlecell.broadinstitute.org/single_cell/app/genes?type=study&page=1&genes=AGPAT&genePage=1) returns no results -- a functional issue with using this as an example.  

Gene NF2, which has results and interacts with PTEN, would fit better:
```suggestion
  let geneSearchPlaceholder = 'genes (e.g. "PTEN NF2")'
```

**Review by eweitz — CHANGES_REQUESTED** [review]
I found somewhat of a functional issue while testing; details below.

**Review by bistline — APPROVED** [review]

**Review by eweitz — APPROVED** [review]

**Pull request merged**